modules/ingest-common/src/test/resources/rest-api-spec/test/ingest/220_drop_processor.yml

```diff
@@ -91,4 +91,4 @@ teardown:
     get:
       index: test
       id: 3
-- match: { found: false }
```

Comment:
This is removed because with this change the `test` index is no longer created since the create index only happens after ingest (and this drops the index request, thus no index ).